procedures/CodeBrowser.ipf

```diff
@@ -943,6 +981,46 @@ Function/S getFunctionLine(lineNo, funcInfo)
 	return StringFromList(lineNo, context, "\r")
 End
 
+// return only fully-commented lines from the given input
+Function/S TrimFunctionComment(context)
+	string context
+
+	string comment = GrepList(context, "^(?i)//.*$", 0, "\r")
+	return RemoveEnding(comment, "\r")
+End
+
+// add basic html
+Function/S AddHTML(context)
+	string context
+
+	string line, html, re
+	string str0, str1, str2, str3, str4
+	variable n, lines
+
+	html = ""
+	lines = ItemsInList(context, "\r")
+	for(n = 0; n < lines; n += 1)
+		line = StringFromList(n, context, "\r")
+		if(GrepString(line, "(?i)\/\/\/.*")) // Doxygen comments
+			re = "(?i)(.*)(@param(?:\[(?:in|out)\])?)\s(\w+)\s(.*)"
+			SplitString/E=(re) line, str0, str1, str2, str3
+			if(V_flag == 4)
+				sprintf line, "%s%s <b>%s</b> %s", str0, str1, str2, str3
```

Comment:
sprintf is limited to x characters depending on the igor version. You have to use `+=` unfortunately. Try opening `MIES_WaveDataFolderGetters` for example.

Reply:
is this relevant for only one line?

Reply:
And line can be of undefined length. Got it.

Reply:
Yes exactly.